landing_page/spec/controllers/user_controller_spec.rb

```diff
@@ -0,0 +1,124 @@
+require 'rails_helper'
+
+RSpec.describe UsersController, type: :controller do
+
+    context "create a new user account with acceptable credentials" do
+
+        it "successfully adds a new user" do
+            expect{create_new_user}.to change(User, :count).by(1)
+        end
+
+        it "logs the user in upon account creation" do
+            create_new_user
+            expect(session[:user_id]).to_not be_nil
+        end
+
+        it "redirects a new user to their profile page upon creation" do
+            create_new_user
+            expect(response).to redirect_to(assigns(:user))
+        end
+
+    end
+
+    context "FactoryBot tests" do
```

Comment:
Context descriptions should be about the stuff you are testing not about the testing / mocking frameworks. You could delete this context and move the factory creation to line 4 after `RSpec.describe UsersController ...`

Reply:
When I did that, it caused all of my tests to fail, including the first three that don't use it as it conflicted with the create_new_user method. I can change the description of the context from "FactoryBots test" to something more appropriate

Reply:
ok one sec , I'll see if I can fix

Reply:
Resolved in commit 5c9ac03bcfe8b27ff22e3b5a6fd61a93633dbafe